lib/index.ts

```diff
@@ -3,20 +3,29 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as _ from 'lodash';
 
-export {
-  buildDepTree,
-  buildDepTreeFromFiles,
-};
+enum DepType {
+  prod = 'prod',
+  dev = 'dev',
+}
 
 interface PkgTree {
   name: string;
   version: string;
   dependencies?: {
     [dep: string]: PkgTree;
   };
+  depType?: DepType;
```

Comment:
`depType` is not currently used by other modules .. we can leave it for feature-parity with `snyk-resolve-deps`.. but don't have to

Reply:
I believe we shouldn't look at this library through the optic of other applications. It should make sense as a standalone module, therefore I'd keep an info whether dependency is `prod` or `dev`.

We can definitely discuss the right naming or structure (e.g. should we follow `package-lock.json` and use `"dev": true,`?), but I'd like to keep an info there.

Reply:
Please lets keep it is it handy for debugging where each dep came from even if it is not yet used anywhere, I see this being useful to show in registry actually against each vuln/dep and would like to suggest we add it there soon